### DISCUSSION TITLE
Running Carla in the Beast (and potentially the Beauty)

### DISCUSSION THREAD

**Discussion by Jokua** [Monash-Connected-Autonomous-Vehicle/autoware#29] [General]
Hi guys @Monash-Connected-Autonomous-Vehicle/avp-team, I noticed that there is a shite ton of docker containers for CARLA in the Beast (and probably in the Beauty), and it's getting a bit hard to keep track of which one is which unless you've bothered naming the containers. I've installed visual studio code with a docker extension in it that lists all of the existing containers, so when you want to run carla, please don't run it via the docker run command UNLESS you have some sort of custom configuration because there's already one or more existing.

You can just go into visual studio code (showing mine for reference as of writing this I am not on the Beast) and click on the docker icon on the left
![image](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/assets/47382093/6baa6a24-3ec6-47df-a756-098e02849258)

Then right click the container that has carla in it and hit start. The container should show up.

Sorry if this sounds a bit like a rant LMAO

**Comment by dylan-gonzalez** [comment]
Were there a lot for Autoware? Shouldn't be, because I think we have been running it with the `rm` flag

**Reply by Jokua** [reply]
not too much, but there's a few duplicates (the top one named headless chicken is the "headless" CARLA container i made for testing with autoware zenoh)

<img width="628" alt="image" src="https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/assets/47382093/61f6e752-f940-462e-9d05-f862db727b80">

some haven't been opened in a month LOL, not touching them because not sure if anyone else has made a container with a specific vers that they need.
